packages/polyml/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.polyml.org
+TERMUX_PKG_DESCRIPTION="Poly/ML is a Standard ML implementation originally written in an experimental language called Poly."
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_VERSION=5.8
+TERMUX_PKG_SRCURL=https://github.com/polyml/polyml/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=6bcc2c5af91f361ef9e0bb28f39ce20171b0beae73b4db3674df6fc793cec8bf
+TERMUX_PKG_DEPENDS="libelf,libffi"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--with-system-libffi --with-pic ac_cv_host=$TERMUX_ARCH-generic-linux-gnu"
+TERMUX_PKG_BLACKLISTED_ARCHES="arm, i686, x86_64"
```

Comment:
What was the reason for blacklisting these architectures?

Reply:
The x86_64 arch has `./polyimport: line 117: /home/builder/.termux-build/polyml/src/.libs/polyimport: No such file or directory`, but it didn't have in aarch64.

Reply:
Reason for that is exactly same as for https://github.com/termux/termux-packages/pull/5528#pullrequestreview-448484220.

The difference is only that target arch is same as for host, but ABI as well as linker path are different. Host environment does not provide the /system/bin/linker64 and thus it errors with "No such file or directory".

If you fix issue I described in https://github.com/termux/termux-packages/pull/5528#pullrequestreview-448484220, this will be fixed as well.